lib/imgresizer.js

```diff
@@ -157,9 +157,16 @@ function processImage(params, method, callback) {
     // normalize query params
     var width = params.w || '';
     var height = params.h || '';
+    var x = params.x || 0;
+    var y = params.y || 0;
     var quality = params.q || 69; // 92 was the default but based on CL-4237 we decided to reduce the quality of jpeg compression for faster load
     var overlay = params.overlay;
-    var filename = sprintf('%s_%s_q%s_%sx%s', path.basename(downloaded_image), method, quality, width, height); // <img-name>_<method>_q<quality>_<width>x<height>
+    var gravity = params.gravity;
```

Comment:
NIT: sometimes gravity can end up being undefined and is still referenced as 
`-gravity undefined` 
in the convert call, but the cropping doesn't seem affected